Resources/Prototypes/Catalog/Fills/Items/briefcases.yml

```diff
@@ -38,5 +38,13 @@
   - type: StorageFill
     contents:
       - id: ClothingEyesGlassesSunglasses
-      - id: SpaceCash1000
-        amount: 5
+      - id: SpaceCash5000
+        amount: 6
+      - id: EncryptionKeySyndie
```

Comment:
this case is used in one of the thief's kits. Please make an alternative version of the suitcase with contents for the thief that does not contain the encryption key and other stuff.

For the thief, the suitcase is part of the communicator's kit, and there implied purely as an element of negotiation. There should be nothing syndicate-related in it.